docs/ht_ww.md

```diff
@@ -1,9 +1,53 @@
 # OVOS Listener - WakeWords / HotWords
-OVOS uses "wakewords" to activate the system.  This is what "hey Google" or "Alexa" is on proprietary devices.  By default, OVOS uses the WakeWord "hey Mycroft".
+OVOS uses "wakewords" to activate the system. This is what "hey Google" or "Alexa" is on proprietary devices. By default, OVOS uses the WakeWord "hey Mycroft".
+
+OVOS "hotwords" is the configuration section in `mycroft.conf` config  to specify what the WakeWord do.  Multiple "hotwords" can be used to do a variety of things from putting OVOS into active listening mode, a WakeWord like "hey Mycroft", to issuing a command such as "stop" or "wake up"
```

Comment:
Grammar is not quite correct here, and in several other places.
I will refer @mikejgray  to that though.

Reply:
Suggestion:

In OVOS, a WakeWord is configured in the `hotwords` section of the user's `mycroft.conf` configuration (defaults to `~/.config/mycroft/mycroft.conf`). Multiple "hotwords" can be configured at one time. They can trigger a variety of tasks, ranging from putting OVOS into active listening mode (the default for "hey Mycroft") to issuing a command such as "stop" or "wake up."

Reply:
Thanks both! I tried to stay close to the original text and only add some extra info. I've followed your suggestion @mikejgray.